src/app/core/components/sobv-page-not-found/sobv-page-not-found.component.html

```diff
@@ -1 +1,19 @@
-<p>sobv-page-not-found works!</p>
+<div class="container">
+
+  <div class="not-found">
+    <div class="blank-page__container">
+      <svg xmlns="http://www.w3.org/2000/svg" width="200" height="200" viewBox="0 0 200 200" fill="none">
```

Comment:
If you need some icons use Bootstrap Icons

Reply:
ok